content/_changelogs/10.7.0.md

```diff
@@ -0,0 +1,81 @@
+## 10.7.0
+
+_Released 8/30/2022_
+
+**Features:**
+
+- Re-introduces the experimental
```

Comment:
@mike-plummer I remember mention of a blog post for this? If we have one, we should call it out. Here is an 10.3.0 is an example where we do this: https://docs.cypress.io/guides/references/changelog#10-3-0

Reply:
Pinged Jess to see about the potential blog post, waiting...

Reply:
Added blog post link from Jess